Metric/Metric.php

```diff
@@ -175,6 +175,9 @@ private function getMetricTags(): string
         if (!empty($this->configurationTags) && is_array($this->configurationTags)) {
             $tags += $this->configurationTags;
         }
+
+        // @todo: essayer resolver symfony pour récupérer des valeurs de service
```

Comment:
:dancer: In english with an issue number if it's gonna stay here ;)

Reply:
It was TODO note for myself. It will be removed.